### ISSUE TITLE
Include NCEI Accession ID and collection DOI/citation in the deployment metadata in ATN's data catalog

### ISSUE THREAD

**Issue by laurabrenskelle** [ioos/marine_life_data_network#33]
### Who is requesting this?

@ioos/marine-life

### What is being requested?

Since data in the ATN data catalog can now be found at multiple places (DataONE, NCEI, OBIS, and GBIF), we need to ensure all of the different representations of data are cross-linked. This issue is intended to track progress on adding appropriate links/citations to the metadata records in the ATN data catalog through an **automated process**. I believe DataONE links are already available in the ATN data catalog. My recommended priority list is:

1. Adding NCEI collection level DOI to all relevant deployment metadata.
2. Adding NCEI Accession link to all relevant deployment metadata
3. Adding OBIS-IPT link to all relevant deployment metadata
4. Adding OBIS link to all relevant deployment metadata
5. Adding GBIF link to all relevant deployment metadata

This should be built as an automated process, not something to be done manually.

### What is the requested deadline and why?

2026-09-30 - this will probably take some time to update the records in the ATN data catalog.

### What is the current status quo (i.e., what happens if this does not get done)?

Users may be unable to follow the thread from the ATN data catalog to the raw data that is stored by NCEI and the synthesized data at OBIS/GBIF.

### What indicates this is done (i.e., how do we know this is complete)?

- [ ] All deployments that have been archived at NCEI include the citation for the collection level record in the ATN data catalog. See [10.25921/wp4e-ph20](https://doi.org/10.25921/wp4e-ph20)
- [ ] All deployments that have been archived at NCEI, include the individual NCEI accession links in the ATN data catalog. See https://www.ncei.noaa.gov/metadata/granule/geoportal/?from=0&size=10&esdsl=%7B%22query%22%3A%7B%22bool%22%3A%7B%22must%22%3A%5B%7B%22query_string%22%3A%7B%22analyze_wildcard%22%3Atrue%2C%22query%22%3A%22fileid%3AIOOS-ATN-STP.*%22%7D%7D%5D%7D%7D%7D#searchPanel
- [ ] All deployments that have been sent to OBIS-USA, include the appropriate links in the ATN data catalog. See https://ipt-obis.gbif.us/?search=ATN&sort=name&order=asc
- [ ] All deployments that have been sent to OBIS, include the appropriate links in the ATN data catalog. See https://obis.org/organization/25716
- [ ] All deployments that have been sent to GBIF, include the appropriate links in the ATN data catalog. See https://www.gbif.org/dataset/search?project_id=ATN

### Provide a description or any other important information.

It may also be important to include `dwc:occurrenceID` as they relate to occurrences in OBIS/GBIF to facilitate a user's ability to backtrack from an OBIS occurrence to the raw data.

**Comment by mmckinzie** [comment]
This task is on hold until ATN DAC to NCEI pipeline can be fully operationalized and automated, and trajectory data files are being served to NCEI and the ATN collection is established. DOI will link to collection as a whole. Individual deployments will receive unique accession numbers.

**Comment by conniekot** [comment]
Sea Name issue from June 5 meeting notes:
NCEI generated sea names, not matching sea names in netCDF files, also typo in sea name file Axiom had been using previously
-- NCEI sent an updated list of sea names - is this the right file? Only names, does not include polygons
-- Also appears there may be differences in how NCEI and Axiom are assigning sea names
--- NCEI uses all data
--- Axiom only data not flagged as suspect in roll_up flag; May also be further binning data? Chris wasn’t sure how.
-- Issue important to resolve as it impacts title, citation, summary

**Comment by conniekot** [comment]
Acronym issue from June 5 meeting notes:
NCEI recommended we do not list acronyms 
- Update to: National Oceanic and Atmospheric Administration (NOAA) Integrated Ocean Observing System (IOOS), Axiom Data Science, Office of Naval Research (ONR), NOAA National Marine Fisheries Service (NMFS), Wildlife Computers, Argos, IOOS Animal Telemetry Network (ATN)

**Comment by mmckinzie** [comment]
These notes are not related to this particular issue. New notes from Connie should be tracked/documented within issue # 41

**Comment by MathewBiddle** [comment]
Thanks for the additional details @mmckinzie.  This task is to make the NCEI Accession Number and DOI available through the ATN Data Portal's metadata record(s). Let's not conflate it with the development of the pipeline itself. That is another task, as you've linked to above.  Yes, this one is dependent on that task, but a much lower priority at the moment. 

Some things to consider here are:

- How to add the information contained in the new fields (in the ADR) from https://github.com/ioos/marine_life_data_network/issues/43 into the ATN Data Portal's public metadata records.
- Where does that info go in the ISO xml record?
- Where else does it need to show up during data access/discovery?
- How much effort is required for those various activities?
- How much of a priorities are all those additions?
- Are there services that NCEI can provide to make management of these identifiers a little easier?
  - IOOS Glider DAC manages an [NCEI Accession Dashboard](https://gliders.ioos.us/status/ncei/?page=1&all=1) where they track datasets archived at NCEI for the Glider DAC. ATN should discuss, with Glider DAC, how that service is managed and if there are tools/lessons learned to leverage for ATN's purpose.

**Comment by mmckinzie** [comment]
@MathewBiddle Yes, that was my whole point. Connie added and then must have subsequently delated comments related to the development of the pipeline and the template(s) and then also edited an above comment, so it no longer captured my original intent, and now my comments appear not to track or related to this issue. Agree, this is a separate but related issue. No confusion on my part.

**Comment by MathewBiddle** [comment]
I like the idea of having a data dashboard that shows what data lives where, similar to the [NGDAC NCEI Accession Dashboard](https://gliders.ioos.us/status/). Could be a simple .html table presented on a webpage. Let's put our heads together this year to figure out the right solution.

**Comment by MathewBiddle** [comment]
Let's put the dashboard idea on hold for now. I'll create a new issue for that and add it to IceBox. 

For this task, I have updated the description https://github.com/ioos/marine_life_data_network/issues/33#issue-2272077462 to clearly articulate what it is we're looking for. Essentially, we need to have an automated way to update the deployment metadata records in the ATN data catalog with all of the links to the same deployment found in other places (DataONE, NCEI, OBIS-USA, OBIS, GBIF).

We might be able to get away with only linking to the OBIS deployment page because that then references to OBIS-USA and GBIF. But, we should discuss.

@laurabrenskelle Moving to ToDo because I think this is a medium priority. Not sure how hard it will be for @iwensu0313 and team, however.

This would be good to have in place for when the profile data pipeline is built.

**Comment by iwensu0313** [comment]
My understanding has been that it's not a trivial task. But the developer who helped set up the BUFR/NDBC dashboard said we could potenitally track the NCEI archives in a similar manner to determine if NCEI has 'accepted' the submissions in the WAF. But we may still need to manually copy paste the newly created project-level DOI that NCEI creates into our system (e.g. ADR or RW) for incorporation into the portal page.

The break in automation is that NCEI is the one generating the DOI. In the case of DataONE, Tetra Tech generates the DOIs, so we can automatically insert it as a hyperlink, etc. For external linkages, those are all manual. But maybe it won't be that bad to manually enter the project-level external DOI?

Either way, I believe we are blocked until 
- https://github.com/ioos/marine_life_data_network/issues/187
- NCEI beings to generate a DOI for each project package and inserts it somewhere easy to copy/paste for incorporation as an external data access link into the ATN Portal

@MathewBiddle  @mmckinzie

**Comment by MathewBiddle** [comment]
Putting this in Blocked as we are having discussions with NCEI on project level accessions.
